### PR TITLE
Add parent property to informationProtectionLabel entity

### PR DESCRIPTION
**Pull request by tommoser**

### CODE REVIEW

File: api-reference/beta/resources/parentlabeldetails.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  "sensitivity": 1024,
+  "tooltip": "String"
+}
+```
+
+<!-- uuid: 16cd6b66-4b1a-43a1-adaf-3a886856ed98
+2019-02-04 14:57:30 UTC -->
+<!-- {
+  "type": "#page.annotation",
+  "description": "parentLabelDetails resource",
```
Comment (zjohnlin) [inline api-reference/beta/resources/parentlabeldetails.md:59]: Do we need this?

File: api-reference/beta/resources/parentlabeldetails.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+<!-- {
+  "blockType": "resource",
+  "optionalProperties": [
+
+  ],
+  "@odata.type": "microsoft.graph.parentLabelDetails",
+  "baseType": null
+}-->
+
```
Comment (zjohnlin) [inline api-reference/beta/resources/parentlabeldetails.md:42]: Do we need this?

**Review by zjohnlin — APPROVED** [review]

File: api-reference/beta/resources/informationprotectionlabel.md [hunk 3]
```diff
@@ -37,6 +37,7 @@ Describes the information protection label that details how to properly apply a
 | name        | String  | The plaintext name of the label.                                                                |
 | sensitivity | Int32   | The sensitivity value of the label, where lower is less sensitive.                              |
 | tooltip     | String  | The tooltip that should be displayed for the label in a UI.                                     |
+| parent      | Label   | The parent label associated with a child label. Null if label has no parent.
```
Comment (millicentachieng) [inline api-reference/beta/resources/informationprotectionlabel.md:40]: Should the type be `labelDetails`?

File: api-reference/beta/resources/labeldetails.md [hunk 4]
```diff
@@ -13,19 +13,21 @@ Namespace: microsoft.graph
...
+| id          | String                                       | The label ID is a globally unique identifier (GUID).                                                         |
+| isActive    | Boolean                                      | Indicates whether the label is active or not. Active labels should be hidden or disabled in user interfaces. |
+| name        | String                                       | The plaintext name of the label.                                                                             |
+| sensitivity | Int32                                        | The sensitivity value of the label, where lower is less sensitive.                                           |
+| tooltip     | String                                       | The tooltip that should be displayed for the label in a user interface.                                      |
+| parent      | microsoft.informationProtection.parentLabelDetails | The parent label associated with a child label.                                                              |
```
Comment (millicentachieng) [inline api-reference/beta/resources/labeldetails.md:30]: The type should be just `parentLabelDetails`?

File: api-reference/beta/resources/informationprotectionlabel.md [hunk 5]
```diff
@@ -63,7 +64,17 @@ The following is a JSON representation of the resource.
   "isActive": true,
   "name": "String",
   "sensitivity": 1024,
-  "tooltip": "String"
+  "tooltip": "String",
+  "parent": {
```
Comment (millicentachieng) [inline api-reference/beta/resources/informationprotectionlabel.md:68]: Replace the parent property with:
```
  "parent": {"@odata.type": "microsoft.graph.labelDetails" }
```

File: api-reference/beta/resources/informationprotectionlabel.md (hunk 3 above)
Comment (isvargasmsft) [reply api-reference/beta/resources/informationprotectionlabel.md]: Thank you!

File: api-reference/beta/resources/labeldetails.md (hunk 4 above)
Comment (isvargasmsft) [reply api-reference/beta/resources/labeldetails.md]: Thank you very much!

File: api-reference/beta/resources/informationprotectionlabel.md (hunk 5 above)
Comment (isvargasmsft) [reply api-reference/beta/resources/informationprotectionlabel.md]: Solved, thank you!

**Comment by isvargasmsft** [comment]
@Lauragra we are good here now :)

**Review by Lauragra — APPROVED** [review]

**Pull request merged**